apps/ai/src/tools/createBooking.ts

```diff
@@ -66,39 +82,34 @@ const createBooking = async ({
   return "Booking created";
 };
 
-const createBookingTool = (apiKey: string, userId: number) => {
+const createBookingTool = (apiKey: string, userId: number, users: UserList) => {
```

Comment:
injecting user data into function call instead of passing through GPT.
Faster + more secure.